README.md

```diff
@@ -16,11 +16,29 @@ Irmin is an OCaml library for building mergeable, branchable distributed data st
 ### Documentation
 Documentation can be found online at [https://mirage.github.io/irmin](https://mirage.github.io/irmin)
 
+#### Prerequisites Note 
+  Please ensure to install the latest Opam version to have the latest packages. (As of now,opam's version can be equal to or higher versions than '2.0.4').
+  
+  Check the Ocaml version greater or equal to '4.09.0' with the help of command 
```

Comment:
```suggestion
  Check the OCaml version greater or equal to '4.07.0' with the help of command
```

Reply:
Hi, @pascutto , but in my case "4.09.0" only worked for me not "4.07.0"

Reply:
Irmin is definitely compatible with 4.07.0. This is checked in CI which you can see [here](https://ci.ocamllabs.io/github/mirage/irmin/commit/7ee44a966820eb46c674d007ca8b9a21e6104d7b).
If you had an issue installing irmin with OCaml 4.07, I suggest you report this back in the issue you opened before (https://github.com/mirage/irmin/issues/981)